_posts/2018-06-14-aerogear-mobile-services-announcement.md

```diff
@@ -0,0 +1,19 @@
+---
+layout: news
+section: news
+title: Announcing AeroGear Mobile Services on OpenShift
+author: jasonmadigan
+---
+
+Over the past six months, the AeroGear team has been busy building on a new vision for building mobile applications atop of OpenShift. Up until now, the AeroGear project has primarily focused on Push Notifications; we're expanding the suite of services we offer today to include a number of other Mobile Services for use in your applications on OpenShift.
```

Comment:
double use of 'building' reads odd. 'building on a new vision for building'

maybe 'building on a new vision for creating'

Reply:
maybe 
 team has been busy with a new vision for building mobile applications on OpenShift

Reply:
working on a new vision?